### DISCUSSION TITLE
How to add GitHub's default "Source code" artifacts

### DISCUSSION THREAD

**Discussion by dmurat** [jreleaser/jreleaser#78] [Q&A] [answered]
I'm experimenting with JRealeser by creating some draft releases on GitHub. In created artifacts, there are no GitHub's default "Source code (zip)" and "Source code (tar.gz)". How can I add these? Is it something that needs to be configured, or GitHub adds those by default for non-draft releases?

Tnx.

**Comment by aalmiray** [comment, marked as answer]
As far as I can tell draft releases do not include the default artifacts you mention.
